matrix/server.py

```diff
@@ -1724,7 +1728,7 @@ def handle_response(self, response):
 
                     if not ret:
                         self.encryption_queue[room_id].pop()
-                        self.encryption_queue[room_id].appendleft(message)
+                        self.encryption_queue[room_id].appendleft(item)
```

Comment:
This is a change in functionality.  I'm not at all sure what's happening here (retrying encrypting a message at a later time maybe?) but I'm pretty sure that what was happening before was wrong, since `message` is a variable that was defined and used in a loop somewhere way earlier.

Reply:
Messages can only be sent when encryption keys are already shared, so this puts the message back into a queue (we have a bunch of queues since we don't have async/await syntax...) if sending the message fails. That's a great catch and probably the cause of https://github.com/poljar/weechat-matrix/issues/79. 

Can we put this into a separate PR? Purely to preserve the history better and mention that the issue might have been fixed. Rest is ok to merge.

Reply:
Done.  Merge https://github.com/poljar/weechat-matrix/pull/126 first, then this one, and mypy should be up and running again.